tests/components/test.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/experimental-ct-react';
+
+import { TestComponent } from '../TestComponent';
+
+test('load initial rois', async ({ mount }) => {
+  const component = await mount(<TestComponent />);
+  const box1 = component.locator('_react=Box[id="box-1"]');
+  const box2 = component.locator('_react=Box[id="box-2"]');
+  expect(await box1.innerText()).toStrictEqual('box1');
```

Comment:
Please never do `expect(await...` but `await expect(...).toHaveText()`

It's a general rule not to do that:
https://playwright.dev/docs/test-assertions